encode_utils/tests/test_connection.py

```diff
@@ -169,5 +174,10 @@ def test_2_make_search_url(self):
     query = "search/?assay_title=ChIP-seq&biosample_type=primary+cell&limit=1&organ_slims=blood&type=Experiment"
     self.assertEquals(res,os.path.join(self.conn.dcc_url,query))
 
+  def test_get(self):
+    res = self.conn.get('experiments/ENCSR502NRF/', frame='object')
```

Comment:
The record you are getting can even be specified more simply ,i.e.
```conn.get('ENCSR502NRF', frame='object')```

Reply:
for some reason this bounced, I think it wasn't following redirect properly.